mkdocs.yml_template

```diff
@@ -12,174 +12,172 @@ nav:
+          - "Configuration": manuals/core/versioning/configuration.md
+          - "Development mode": manuals/core/versioning/development_mode.md
+          - "Experiment mode": manuals/core/versioning/experiment_mode.md
+          - "Production mode": manuals/core/versioning/production_mode.md
+        - "Scheduling":
+          - manuals/core/scheduling/index.md
       - "Taipy REST":
-          - manuals/rest/index.md
+        - manuals/rest/index.md
       - "Authentication and Roles":
-          - "Authentication and Authorization": manuals/auth/index.md
-          - "Authentication": manuals/auth/authentication.md
-          - "Authorization": manuals/auth/authorization.md
+        - "Authentication and Authorization": manuals/auth/index.md
+        - "Authentication": manuals/auth/authentication.md
+        - "Authorization": manuals/auth/authorization.md
       - "Taipy command-line interface":
-          - manuals/cli/index.md
-          - "help": manuals/cli/help.md
-          - "create": manuals/cli/create.md
-          - "run": manuals/cli/run.md
-          - "manage-versions": manuals/cli/manage-versions.md
-          - "version": manuals/cli/version.md
+        - manuals/cli/index.md
+        - "help": manuals/cli/help.md
+        - "create": manuals/cli/create.md
+        - "run": manuals/cli/run.md
+        - "manage-versions": manuals/cli/manage-versions.md
+        - "version": manuals/cli/version.md
       - "Running & Deploying":
-          - "Running and Deploying Taipy applications": manuals/run-deploy/index.md
-          - "Running":
-              - "Running a Taipy application": manuals/run-deploy/run/index.md
-              - "The run() function": manuals/run-deploy/run/running_services.md
-              - "Using an external web server": manuals/run-deploy/run/external_web_server.md
-              - "Protect private files": manuals/run-deploy/run/protect_files.md
-          - "Deployment":
-              - "Deploying a Taipy application": manuals/run-deploy/deploy/index.md
-              - "On the local machine":
-                  - "Deploying locally": manuals/run-deploy/deploy/local/index.md
-              - "On Linux servers":
-                  - "Deploying on a Linux system": manuals/run-deploy/deploy/linux/index.md
-                  - "Ubuntu": manuals/run-deploy/deploy/linux/ubuntu.md
-                  - "RHEL": manuals/run-deploy/deploy/linux/redhat.md
-              - "On Windows Server":
-                  - "Deploying on Windows Server": manuals/run-deploy/deploy/windows/index.md
-              - "Using Docker":
-                  - manuals/run-deploy/deploy/docker/index.md
-                  - "For development": manuals/run-deploy/deploy/docker/development.md
-                  - "For production": manuals/run-deploy/deploy/docker/production.md
-              - "On Azure":
-                  - "Deploying on Azure": manuals/run-deploy/deploy/azure/index.md
-  #            - "On Clever Cloud":
```

Comment:
Why remove that? It could have been useful in the future... or not.

Reply:
Why not... I don't mind.